release.sh

```diff
@@ -0,0 +1,330 @@
+#!/bin/bash
+red=`tput setaf 1`
+green=`tput setaf 2`
+yellow=`tput setaf 3`
+blue=`tput setaf 4`
+reset=`tput sgr0`
+
```

Comment:
If possible, try to use the `ps` function to discover the command line name, so this way this will have the name not hardcoded into script.

`basename` is the function you might be looking for: http://stackoverflow.com/questions/192319/how-do-i-know-the-script-file-name-in-a-bash-script